scripts/.phantom-run-tests.js

```diff
@@ -0,0 +1,41 @@
+var page = require('webpage').create();
+
+// Must match variable of the same name below.
+var logToken = 'e54b0a2b-0934-400f-bf3a-68d01dcde2df';
+var someTestsFailed = false;
+
+function installConsoleListener() {
+  page.onConsoleMessage = function(msg) {
+    if (msg.substring(0, logToken.length) == logToken) {
+      var commandString = msg.substring(logToken.length);
+      if(commandString == 'fail') {
+        someTestsFailed = true;
+      }
+    } else {
+      console.log(msg);
+    }
+  };
+}
+
+page.open('http://server/', function(status) {
+  installConsoleListener();
+  page.evaluate(function() {
+    var logToken = 'e54b0a2b-0934-400f-bf3a-68d01dcde2df';
```

Comment:
Let me know if I'm confused, but take a look at the scoping. The previous definition is at the root scope of this file. It should be available here.